modelzoo/unet/scripts/run_dist.sh

```diff
@@ -15,31 +15,36 @@
 #!/bin/bash
 set -x
 
+echo "############################## setting env ##############################"
+master="vsr214"
+hosts="vsr214,vsr215,vsr216,vsr218"
```

Comment:
Hostname should not appeared, please use args to input.

Reply:
fixed